### DISCUSSION TITLE
Easy question: Set default theme

### DISCUSSION THREAD

**Discussion by hairlesshobo** [magicbug/Cloudlog#743] [Q&A] [answered]
I know that you can choose the dark or light theme as the default theme for a particular user. Is there a way to select the dark theme as default for non-authenticated users as well? I looked though the config.php file and didn't see anything, but thought maybe there was an undocumented setting for it or something.

Thanks!

**Comment by magicbug** [comment, marked as answer]
Note changed this into a discussion as it's not an issue with code

Not currently, will be coming when #630 is finalised and added to the master branch.

**Reply by hairlesshobo** [reply]
Sorry for posting it in the wrong place. Still fairly new to using Github.

Thanks for the quick reply. You guys are awesome!
